tfs/src/main/java/hudson/plugins/tfs/model/ChangeLogSet.java

```diff
@@ -19,16 +21,26 @@
 
     private final List<ChangeSet> changesets;
 
+    @Deprecated

```

Comment:
A TODO here with information on when we should remove these deprecated methods would help. For instance, should we remove them with the next major version release or can they be removed with the next minor release. A comment on who might be using these methods would also help clarify when they can be removed.

Reply:
One of the methods is still being used by a few tests (added a comment). Removed the other.